src/components/Navigation.vue

```diff
@@ -5,14 +5,19 @@
 				to="/"
 				class="text-white text-2xl font-bold flex items-center"
 			>
-				<img class="h-10 mr-3 rounded-full" src="../assets/img/logo.png">
+				<img
+					class="h-10 mr-3 rounded-full"
+					src="../assets/img/logo.png"
+				/>
 				AntiPlag
 			</router-link>
+			<router-link to="/upload" class="text-white">
```

Comment:
We need to fix this navigation later...  it's broken... but it's out of scope of this task.